tests/components/homekit_controller/test_sensor.py

```diff
@@ -76,3 +87,17 @@ def create_light_level_sensor_service():
     helper.characteristics[LIGHT_LEVEL].value = 20
     state = await helper.poll_and_get_state()
     assert state.state == "20"
+
+
+async def test_carbon_dioxide_level_sensor_read_state(hass, utcnow):
+    """Test reading the state of a HomeKit carbon dioxide sensor accessory."""
+    sensor = create_carbon_dioxide_level_sensor_service()
+    helper = await setup_test_component(hass, [sensor], suffix="carbon_dioxide_level")
```

Comment:
I ran the tests locally and actually suffix is what gets added to the entity name after `velux_sensor_` - so it actually needs to be co2.

With the formatting changes i already commented + this test fix ^^ all the tests and lints are passing locally.